### DISCUSSION TITLE
Soroban Guide by Tommaso

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#79] [General]
Given that there is no info on soroban events on the docs (and on the examples it seems too) since they are still in early development, we have decided to write a simple contract + blog post about what events are how they work (compared to Solidity), how to use them in a soroban smart contract and how to read them from Horizon.

[Repo](https://github.com/xycloo/soroban-events-guide)
Post: https://heytdep.github.io/comp_posts/95)--pub)--Playing-with-Soroban-events/post.html

I (@heytdep) have also updated [my blog](https://heytdep.github.io/) to hold everything we are writing for the Sorobanathon (I should maybe have a custom tag for that lol).

**Comment by heytdep** [comment]
We also thought that it might be a good idea to group and organize all the soroban resources we wrote, so we have created a new repo for that (https://github.com/xycloo/soroban-guide).

**Comment by achallagundla** [comment]
Heya @heytdep ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 2000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
